tests/e2e-kubernetes/manifest.yaml

```diff
@@ -0,0 +1,31 @@
+# Manifest for Kubernetes external tests.
+# See https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external
+
+ShortName: ebs
+StorageClass:
+  FromFile: storageclass.yaml
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: ebs.csi.aws.com
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 16Ti
+  SupportedFsType:
+    xfs: {}
+    ext4: {}
```

Comment:
add ext2 and ext3 since they are supported too

Reply:
I am not sure it brings any real value. It only doubles number of tests. It's served by ext4 kernel module anyway.